atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -422,12 +410,6 @@ public void timeLockMigrationReportsReadyIfMigrationNotNeeded() {
         assertTrue(TransactionManagers.timeLockMigrationCompleteIfNeeded(lockAndTimestampServices));
     }
 
-    @Test
-    public void usesTimeLockIfInstallConfigIsUnspecifiedButInitialRuntimeConfigContainsTimeLockBlock() {
-        setUpTimeLockBlockInRuntimeConfig();
-        verifyUsingTimeLockByGettingAFreshTimestamp();
-    }
-
     @Test
     public void throwsIfInstallConfigHasLeaderBlockButInitialRuntimeConfigContainsTimeLockBlock() throws IOException {
```

Comment:
Would like to know what happens if leader block is there and timelock block has zero servers.

Reply:
OK, yes - that wasn't clear immediately, but the timelockRuntime block is there and empty.